README.md

```diff
@@ -2,35 +2,49 @@
-2. _alternate_ documents; these contain supporting definitions for a Who's On First place and are named `[WOFID]-alt-*.geojson` (currently `[WOFID]-alt-quattroshapes_pg.geojson` and `[WOFID]-alt-naturalearth.geojson` exist in the `whosonfirst-data` repo)
+## Schema Types
 
-## JSON Schema Version
+This repo contains JSON schema definitions for the following JSON file types:
 
-The schemas in this repo have been written to conform to [JSON Schema v0.6 draft](http://json-schema.org/specification-links.html#draft-6) (note to self: when _will_ JSON Schema ever reach version 1.0 and come out of draft?).
+### Who's On First Property Definitions
+
+`schema/properties/properties.json` defines the JSON schema to validate Who's On First property definition files, found in the [`whosonfirst-properties`](https://github.com/whosonfirst/whosonfirst-properties) repo.
+
+### GeoJSON Documents
+
+`schema/geojson/geojson.json` defines the JSON schema to validate a WOF specific subset of GeoJSON documents.
 
-## Schema documents
+### Who's On First Documents
 
-This repo contains the following top level schemas:
+`schema/docs/whosonfirst.json` defines the JSON schema to validate Who's On First GeoJSON documents, found in the [`whosonfirst-data`](https://github.com/whosonfirst-data/whosonfirst-data) repo and is a superset of the GeoJSON schema.
+
+## Referenced Schema Definitions
+
+In addition to the top level schemas detailed above, this repo also contains the following _referenced_ (JSON schema-speak for _included_) schema definitions:
+
+* `schema/docs/wof-properties.json` - defines a Who's on First document's `properties` object
+
+* `schema/docs/wof-hierarchy.json` - defines a Who's On First `wof:hierarchy` object
+
+* `schema/geojson/geoson-bbox.json` - defines a GeoSON `bbox` object
+* `schema/geojson/geojson-geometry.json` - defines a GeoJSON `geometry` object
+
+## Who's On First Document types
+
+At the time of writing, the [`whosonfirst-data`](https://github.com/whosonfirst-data/whosonfirst-data) repo contains two types of GeoJSON documents :-
 
-* `schema/geojson.json` - validates a GeoJSON document
-* `schema/whosonfirst.json` - validates a _master_ Who's On First GeoJSON document
-* `schema/whosonfirst-alt.json` - validates an _alternate_ Who's On First GeoJSON document
+1. _master_ documents; these contain the canonical definition of a Who's On First place and are named `[WOFID].geojson`. These are validated by the schema contained in `schema/docs/whosonfirst.json`.
 
-This repo also contains the following referenced schemas:
+2. _alternate_ documents; these contain supporting definitions for a Who's On First place and are named `[WOFID]-alt-*.geojson` (currently `[WOFID]-alt-quattroshapes_pg.geojson` and `[WOFID]-alt-naturalearth.geojson` exist in the `whosonfirst-data` repo). These are validated by the schema contained in `schema/docs/whosonfirst-alt.json`.
```

Comment:
Noting that we have the option to add function/extras to the alt-geometry filename: https://github.com/whosonfirst/whosonfirst-cookbook/blob/master/how_to/creating_alt_geometries.md#what-optional-rules-apply-to-alt-geometries

See: 
- https://github.com/whosonfirst-data/whosonfirst-data/blob/master/data/859/225/83/85922583-alt-mapzen-land.geojson
- https://github.com/whosonfirst-data/whosonfirst-data/blob/master/data/856/886/37/85688637-alt-uscensus-display-terrestrial-zoom-10.geojson

Reply:
@vicchi - yep, that works.

The changes in your diff are not wrong, I just think it may be useful to expand 

`[WOFID]-alt-*.geojson`

to

`[WOFID]-alt-source-function-scope-detail.geojson`

Reply:
Fixed; see commit `f47bc88`